views/common.twig

```diff
@@ -525,20 +525,8 @@
                     save: "{{ "Save" |trans }}",
                     cancel: "{{ "Cancel" |trans }}",
                 },
-                positionTab: {
```

Comment:
There was a merge conflict here, we need to keep the `positionTab` properties

Reply:
Strange, I checked the copy of westphal and it doesn't have the `positionTab` in it. I'll have this added back.